lms/djangoapps/student_account/views.py

```diff
@@ -133,9 +133,7 @@ def login_and_registration_form(request, initial_mode="login"):
             'third_party_auth_hint': third_party_auth_hint or '',
             'platform_name': configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME),
             'support_link': configuration_helpers.get_value('SUPPORT_SITE_LINK', settings.SUPPORT_SITE_LINK),
-            'password_reset_support_link': configuration_helpers.get_value(
-                'PASSWORD_RESET_SUPPORT_LINK', settings.SUPPORT_SITE_LINK  # Intentional default.
-            ),
```

Comment:
Thanks @UmanShahzad. Please rebase https://github.com/edx/configuration/pull/3888 when you get a chance. I will take care of edx/edx-internal.